rs_workflows/staging.py

```diff
@@ -580,16 +356,16 @@ def __init__(  # pylint: disable=too-many-arguments
         """
         Initialize the PrefectFlowConfig object with provided parameters.
         """
-        super().__init__(user, url, url_catalog, station, mission, tmp_download_path, s3_path, apikey)
+        super().__init__(rs_client, mission, tmp_download_path, s3_path)
         self.max_workers: int = max_workers
         self.start_datetime: datetime = start_datetime
         self.stop_datetime: datetime = stop_datetime
         self.limit = limit
 
 
 @flow(task_runner=DaskTaskRunner(cluster_kwargs={"n_workers": 15, "threads_per_worker": 1}))
```

Comment:
@agrosu1978 it seems to me here that we have the same information twice: the number of workers passed to prefect (the `n_workers` parameter) is the same than `PrefectFlowConfig.max_workers` ? 

So maybe we could remove `PrefectFlowConfig.max_workers` and take the information from `n_workers` instead ? (if we can retrieve this parameter value from the python code ?)

Reply:
Prefect creates a temporary local cluster automatically with the number of workers used based on the number of cores on your machine. I don't know how we are going to use the Dask cluster on our cluster, is this going to be used locally on the pod, or a specific one deployed on our cluster? So, the n_workers is the maximum number of tasks that can be running in parallel. The max_workers is the number of tasks that our flow can create, but if this is higher than n_workers, of course that only n_workers will run in parallel, while the (max_workers - n_workers) tasks wait for a running tasks to finish in order to take that slot.
I noticed that using n_workers with a high value, simply blocks the PC. Anyway, we can see in the future after some experimentation with prefect flows if in fact the max_workers may be the same as n_workers. Let's keep it like this for now